<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWidget</class>
 <widget class="QWidget" name="CalibrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1294</width>
    <height>844</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calibration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="7">
    <widget class="QPushButton" name="startButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Start Calibration</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="updateButton">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QPushButton" name="stopShowing">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Stop Showing</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="videoBox"/>
   </item>
   <item row="1" column="9">
    <widget class="QPushButton" name="saveCalib">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Save Calibration</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QSpinBox" name="heightSpinBox">
     <property name="value">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QSpinBox" name="widthSpinBox">
     <property name="value">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>height</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="inputSettingsButton">
        <property name="text">
         <string>Input</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../resources/main.qrc">
          <normaloff>:/new/prefix1/webcam.png</normaloff>:/new/prefix1/webcam.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="camerasSettingsButton">
        <property name="text">
         <string>Settings</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../resources/main.qrc">
          <normaloff>:/new/prefix1/2cam-blend-b.png</normaloff>:/new/prefix1/2cam-blend-b.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startRecordingButton">
        <property name="toolTip">
         <string>Start Recording</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../resources/main.qrc">
          <normaloff>:/new/prefix1/play.png</normaloff>:/new/prefix1/play.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopRecordingButton">
        <property name="toolTip">
         <string>Stop Recording</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../resources/main.qrc">
          <normaloff>:/new/prefix1/stop_2.png</normaloff>:/new/prefix1/stop_2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseRecordingButton">
        <property name="toolTip">
         <string>Pause Recording</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../resources/main.qrc">
          <normaloff>:/new/prefix1/pause.png</normaloff>:/new/prefix1/pause.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="11">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1218</width>
        <height>766</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QGroupBox" name="imageBox">
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Images</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="6" column="11">
    <widget class="QWidget" name="widget" native="true"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startRecordingButton</sender>
   <signal>released()</signal>
   <receiver>CalibrationWidget</receiver>
   <slot>startRecording()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateButton</sender>
   <signal>released()</signal>
   <receiver>CalibrationWidget</receiver>
   <slot>updateVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>released()</signal>
   <receiver>CalibrationWidget</receiver>
   <slot>startCalibration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopShowing</sender>
   <signal>released()</signal>
   <receiver>CalibrationWidget</receiver>
   <slot>stopShowing()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveCalib</sender>
   <signal>released()</signal>
   <receiver>CalibrationWidget</receiver>
   <slot>saveMatrix()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>811</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addImage()</slot>
  <slot>startCalibration()</slot>
  <slot>startRecording()</slot>
  <slot>updateVideo()</slot>
  <slot>stopShowing()</slot>
  <slot>saveMatrix()</slot>
 </slots>
</ui>
